src/controllers/userController.js

```diff
@@ -11,7 +11,33 @@ const firebaseErrors = {
 const getProfile = async (req, res) => {
   const { id } = req.params
   try {
-    const userProfile = await UserService.getProfile(id)
+    let userProfile = await UserService.findById(id)
+
+    //TODO: Need to find out why is not returning userProfile
```

Comment:
@hebertporto , poderia dar uma olhada nisso depois?
Nao sei o que ta acontecendo, o `findById` sempre retorna null.

To dando algum vacilinho, ☹️